tests/playbooks/tests_eth_dns_support.yml

```diff
@@ -108,7 +108,7 @@
           - "'timeout:1' in ipv6_dns.stdout"
         msg: "DNS options are configured incorrectly"
 
-- import_playbook: down_profile.yml
+- import_playbook: down_profile+delete_interface.yml
```

Comment:
The test is removing the interface already at the end, so it seems this is now done twice.

Reply:
This is happening in multiple tests. Fixing it.

Reply:
Fixed! Thanks!